### ISSUE TITLE
Expose Denied exceptions from platform-specific part to common

### ISSUE THREAD

**Issue by Alex009** [icerockdev/moko-permissions#4]
Denied exceptions now in platform-specific code. Should be in common code, for checking like:
```
try {
    permissionsController.providePermission(Permission.CAMERA)
} catch(error: DeniedException) {
    // user denied permission request
} catch(error: DeniedAlwaysException) {
    showPlaceholder = true
}
```

**Comment by Alex009** [comment]
@Tetraquark can you add samples of usage both excaptions?
show on UI placeholder "you not have permission, grant it" if we got `DeniedException` and "permissions disabled, please enable it in settings" if we got `DeniedAlwaysException`

**Comment by Tetraquark** [comment]
The example of using PermissionsController class and exceptions handling will be implemented in #2 issue (first implementation c667910)

**Comment by Alex009** [comment]
already done by @Tetraquark , thx
